configure

```diff
@@ -233,7 +233,7 @@ show()
   case "$*" in
     *$test.c*)
       echo "=== $test.c ===" >> configure.log
-      cat $test.c >> configure.log
+     cat $test.c >> configure.log
```

Comment:
Looks like whitespace change?

Reply:
Hmm, not sure how that managed to sneak in.  Easy fix, though.